1_space_odyssey/js/mannuel-9000/ben-9000/app.js

```diff
@@ -0,0 +1,19 @@
+var isAlive = true; 
+var name = "ben-9000";
+var hasJetPack = true;
+var suitColor = "black";
+var eyes = 4;
+var eyeColor = "brown";
+var hairColor = 0;
+var legs = 2 ;
+var arms = 2;
+var hands = 2;
+var ears = 2;
+var canWalk = true;
+var canRun = true;
+var canFly = true;
+var hasWeapon = false;
+var isInGoodMood = false;
+
+console.log(".-.-. Booting .-.-.");
+console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
```

Comment:
please do exercise 2. as well.

its this one: https://github.com/mannuelf/feu1_programming_foundations/blob/master/1_space_odyssey/js/mannuel-9000/app.js#L111

![https://media.giphy.com/media/S98vo46fyH2PS/giphy-downsized.gif](https://media.giphy.com/media/S98vo46fyH2PS/giphy-downsized.gif)